library/sinks/Fetch.test.ts

```diff
@@ -140,12 +150,10 @@ t.test(
       {
         ...context,
         // Redirects to http://127.0.0.1/test
-        ...{ body: { image: "https://dub.sh/aikido-ssrf-test" } },
+        ...{ body: { image: redirectUrl.ip } },
```

Comment:
should we move the comments to the def at the top?

Reply:
Moved them.